sys/include/net/gcoap.h

```diff
@@ -258,6 +263,7 @@ extern "C" {
 #ifndef GCOAP_REQ_WAITING_MAX
 #define GCOAP_REQ_WAITING_MAX   (2)
 #endif
+/** @} */
```

Comment:
Can't you group the remaining macros with the once above instead of adding all the macros by hand?

Reply:
I debated about that, but I think `gcoap.h` is easier to read with the macros grouped functionally. For example, the token length macros are grouped together. It just so happens that GCOAP_TOKENLEN is configurable. I also thought the review would be simpler without the extra layer of rearrangement.

Is there a better way to approach this? It seems like a single hierarchy always will be a problem because the best organization depends on the reader's perspective. So, a tagging approach like this seems useful.

Reply:
No, if there is a reasoning behind this, I'm fine with it.

Reply:
OK then, no code change needed.